fulp_modules/_defines/fulp_defines.dm

```diff
@@ -51,6 +41,3 @@
 
 ///Define for the 'Rabbits' Faction.
 #define FACTION_RABBITS "rabbits"
-
-//Define for the Syndicate Engineer Ruin, used in 'syndicate_engineer.dm"
-#define ROLE_SYNDICATE_ENGINEER "Syndicate Engineer"
```

Comment:
What’s the reason for removing the define?

Reply:
We could keep it, its only used in one place so it seems weird to define it for only one use.

Reply:
I think you should keep it in